src/Smalot/PdfParser/Font.php

```diff
@@ -267,6 +267,38 @@ public function setTable(array $table)
         $this->table = $table;
     }
 
+    /**
+     * Calculate text width with data from header 'Widths'. If width of character is not found then character is added to missing array.
+     */
+    public function calculateTextWidth($text, &$missing = null): ?float
```

Comment:
Can we change second parameter to `array &$missing = null`? This way its type is explicit and more clear.

Reply:
Done